### ISSUE TITLE
Crisis: Missing fil translation all over the window today

### ISSUE THREAD

**Issue by jidanni** [openstreetmap/iD#9615]
![20230501T073236](https://user-images.githubusercontent.com/1230959/235381158-5d35b76d-469c-4de1-a942-1126815e9b01.jpg)

**Comment by jidanni** [comment]
OK, pushing the browser's refresh button makes the problem go away.
But it shouldn't happen in the first place.

**Comment by 1ec5** [comment]
What are the preferred languages in your browser or operating system settings? Also, what are your preferred languages in osm.org preferences?

**Comment by jidanni** [comment]
I am assuming as it was the first thing I browsed for the entire day, that some parts of the page just didn't load. So when I refresh it I fixed it.

Okay well I'm going to close this because I guess it could happen to any page at any website.

Or I suppose I could report a browser bug to the particular browser company. But I guess the refresh button is there for reason and maybe 20 years from now we can demand more perfection.

The bug would be: there should be some inspection mechanism to be sure all parts of a page of arrived before displaying it.

But on the other hand HTML is made to accept various failures.

Okay never mind.
